CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+Contributing
+============
+
+It's good to hear that you want to contribute to Corral!
+
+There are a number of ways to contribute to Corral. As this document is a little long, feel free to jump to the section that applies to you currently:
+
+* [Feature request](#feature-request)
+* [Bug report](#bug-report)
+* [How to contribute](#how-to-contribute)
+* [Pull request](#pull-request)
+
+Additional notes regarding formatting:
+
+* [Documentation formatting](#documentation-formatting)
+* [Code formatting](#code-formatting)
+
+Feature request
+---------------
+For any feature requests or enhancements to Corral, it is quite likely that you have to go through our [RFC process](https://github.com/ponylang/rfcs). Before opening or submitting any feature requests, please make sure you are familiar with the RFC process and follow the process as required.
+
+If you submit a pull request to implement a new feature without going through the RFC process, it may be closed with a polite request to submit an RFC first.
+
+Bug report
+----------
+First of all please [search existing issues](https://github.com/ponylang/corral/issues) to make sure your issue hasn't already been reported. If you cannot find a suitable issue — [create a new one](https://github.com/ponylang/corral/issues/new).
+
+Provide the following details:
+
+  - short summary of what you was trying to achieve,
+  - a code causing the bug,
+  - expected result,
+  - actual results and
+  - environment details: at least operating system and compiler version (`ponyc -v`).
```

Comment:
Maybe we should add a version to Corral that is easily available on the command line and request that here too.

Reply:
Agreed. Can you open a PR for that?

Reply:
OK, I will work on that.